### PR TITLE
Feature: default context at t

### PR DESCRIPTION
**Pull request by cap10morgan**
Finishes https://github.com/fluree/core/issues/9

Commits 2d8d451cbcbaf7c3076cb3e091ca176f758aa26e & 01236dc98740008546d6a4bd9935f10c31670346 are unrelated cleanups. Let me know if the latter deserves its own PR for discussion.

I've exposed defaultContext at t via a new, optional `t` param to the `GET /defaultContext` endpoint. This needed a reify fix in db to work, so its pointed at that commit currently. db PR is here: https://github.com/fluree/db/pull/529

### CODE REVIEW

**Review by zonotope — APPROVED** [review]
🪴

**Comment by cap10morgan** [comment]
Merging blocked until db PR is approved and merged

**Pull request merged**
